### PR TITLE
Ophelia/eta

### PR DESCRIPTION
**Pull request by OpheliaLjh**
Hardcoded the order status page, when user checkout the page will show up and display the eta and distance

### CODE REVIEW

File: src/components/Checkout/body.tsx [hunk 1]
```diff
@@ -10,13 +10,30 @@ import { useState } from "react";
 import { ImSpinner3 } from "react-icons/im";
 import { toast } from "react-toastify";
 
+
```
Comment (JunTaoLuo) [inline src/components/Checkout/body.tsx:13]: nit: Why the extra whitespace?

File: src/components/Checkout/body.tsx [hunk 2]
```diff
@@ -10,13 +10,30 @@ import { useState } from "react";
...
 const Body = ({ action }: { action: any }) => {
   const [
-    { checkoutData, cartTotal, paymentMethod, cartItems, DonutItems },
+    { checkoutData, cartTotal, paymentMethod, cartItems, DonutItems, showContactForm, showOrder, showCart },
     dispatch,
   ] = useStateValue();
   const [loading, setLoading] = useState(false);
 
+
+  // const completePayment = () => {
```
Comment (JunTaoLuo) [inline src/components/Checkout/body.tsx:30]: Remove dead code

**Review by JunTaoLuo — APPROVED** [review]
Changes look good, just a few minor nitpicks

**Comment by OpheliaLjh** [comment]
I have cleaned the codes

**Pull request merged**